lib/DDG/Spice/NearEarthObjects.pm

```diff
@@ -0,0 +1,20 @@
+package DDG::Spice::NearEarthObjects;
+# ABSTRACT: Shows details of objects (asteroids) that come close to earth — within several million miles.
+
+use strict;
+use DDG::Spice;
+
+spice is_cached => 1;
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://api.nasa.gov/neo/rest/v1/feed?format=JSON&api_key={{ENV{DDG_SPICE_NASA_APIKEY}}}';
+
+my @triggers = ('near earth object', 'near earth objects', 'near earth asteroid', 'near earth asteroids', 'near-earth object', 'near-earth objects', 'near-earth asteroid', 'near-earth asteroids', 'close asteroid', 'close asteroids');
```

Comment:
Lets pull some of these onto new lines (indented to match first item), then we won't have them going off the screen!